ansible/configs/sap-smart/post_software.yml

```diff
@@ -133,6 +133,28 @@
         mode: '0600'
         force: yes
 
+- name: Prevent Pulp 500 error
+  hosts: sats
+  become: yes
+  tasks:
+
+    - name: Make sure MongoDB will be able to start
+      shell: >
+        echo "-vv" > /.autorelabel &&
+        foreman-maintain service stop &&
```

Comment:
This seems not very ansible-y? Particularly echoing to a file with `shell`. It is also not clear what this shell command has to do with MongoDB? Maybe something like:

```
  - name: Create /.autorelabel to trigger SELinux relabeling
    copy:
      content: -vv
      dest: /.autorelabel
      
  - name: Run this command for some reason...
    command: foreman-maintain service stop
```